jaeger-core/build.gradle

```diff
@@ -19,6 +20,18 @@ dependencies {
     signature 'org.codehaus.mojo.signature:java16:1.1@signature'
 }
 
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
We use this plugin to relocate the okhttp library. This enables consumers of jaeger-core to depend on a different version of okhttp if they choose to.